### ISSUE TITLE
i18n config for yii\i18n\DbMessageSource: 'db' property not working

### ISSUE THREAD

**Issue by adzadzadz** [yiisoft/yii2#9648]
I'm working with multiple databases in a single yii2 application and it seems that the $db property from DbMessageSource isn't working. 

Console: php ./yii message/extract common/config/i18n.php still looking for the 'db' default connection component whereas I already modified:
  ```
   // 'db' output format is for saving messages to database.
    'format' => 'db',
    // Connection component to use. Optional.
    'db' => 'db_glb_sys',
    // Custom source message table. Optional.
    'sourceMessageTable' => '{{%language_source}}',
    // Custom name for translation message table. Optional.
    'messageTable' => '{{%language_message}}',
```

Here's the db id I want the translations to connect to
```
'components' => [
        'db_glb_sys' => [
            'class' => 'yii\db\Connection',
            'dsn' => 'mysql:host=localhost;dbname=glb_sys_db_01',
            'username' => 'root',
            'password' => '',
            'tablePrefix' => 'glb_',
            'charset' => 'utf8',
        ],
```

========================== EDIT ================================

Okay I found real problem. It is in yii2/framework/console/controllers/MessageController.php line 172 to 177. 

```
 protected function saveMessagesToDb($messages, $db, $sourceMessageTable, $messageTable, $removeUnused, $languages, $markUnused)
    {
        $q = new \yii\db\Query;
        $current = [];

        foreach ($q->select(['id', 'category', 'message'])->from($sourceMessageTable)->all() as $row) {
            $current[$row['category']][$row['id']] = $row['message'];
        }
```

The issue is, \yii\db\Query only tries to connect to the core application component 'db'. WHICH I don't want to use in storing my translation data.

**Comment by adzadzadz** [comment]
![screenshot_1](https://cloud.githubusercontent.com/assets/4595687/9780122/fe195dcc-57b6-11e5-9bf5-0aec2cd43bad.png)

**Comment by adzadzadz** [comment]
It seems this has been resolved but not yet ammended in v2.0.6
